ansible/roles/session-valkey/templates/values.yml.j2

```
@@ -0,0 +1,54 @@
+global:
+  defaultStorageClass: '{{ SC_DEFAULT_STORAGE_CLASS_NAME }}'
```

change the ' to " at this line
```suggestion
  defaultStorageClass: "{{ SC_DEFAULT_STORAGE_CLASS_NAME }}"
```